### PR TITLE
Feature/fastlane

### PR DESCRIPTION
**Pull request by JulienArzul**
Added Fastlane and removed some unused dependencies (Qt and Kotlin compiler)

For reference, fix for Fastlane integration on Ubuntu:
fastlane/fastlane#11579

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,9 @@ RUN dpkg --add-architecture i386 && \
     apt-get install -y libncurses5:i386 libc6:i386 libstdc++6:i386 lib32gcc1 lib32ncurses5 lib32z1 zlib1g:i386 && \
     apt-get install -y --no-install-recommends openjdk-8-jdk && \
     apt-get install -y git wget zip && \
-    apt-get install -y qt5-default
+    apt-get install -y ruby-full build-essential dh-autoreconf
+
+RUN gem install fastlane -NV --no-document
```
Comment (sunilchand) [inline Dockerfile:25]: Nice work getting fastlane into linux!

Comment (JulienArzul) [reply Dockerfile:25]: Yeah, it should be easier in the future: they're supposed to be working on a multi-platform support or at least, they've got an issue open for it! (https://github.com/fastlane/fastlane/issues/11687)

For reference, I found the fix here:
https://github.com/fastlane/fastlane/issues/11579

**Review by sunilchand — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -29,17 +31,8 @@ RUN mkdir -p /opt/android-sdk && cd /opt/android-sdk && \
     unzip *tools*linux*.zip && \
     rm *tools*linux*.zip
 
-# download and install Kotlin compiler
-ENV KOTLIN_VERSION 1.2.61
```
Comment (maks-lin-bilue) [inline Dockerfile:33]: So I'm guessing gradle will download Kotlin on demand?

Comment (JulienArzul) [reply Dockerfile:33]: Yes, I'm pretty sure it is actually already doing that by itself and that version of the compiler is never used by the Android build => the Kotlin compiler is a dependency of the Kotlin gradle plugin
(maybe the KOTLIN_HOME variable overrides the normal behaviour but I doubt it)

And it's actually a lot better to not care about the version installed on our build machines and declare it project by project.

Comment (maks-lin-bilue) [reply Dockerfile:33]: 👍

**Pull request merged**